deployment/CE04OSPS_Deploy.csv

```diff
@@ -44,18 +44,33 @@ SKQ201610S,,,,CE04OSPS-SF01B-3D-SPKIRA102,3,1,2016-07-19T08:00:00,2017-07-28T00:
+RR1713,,,,CE04OSPS-PC01B-4A-DOSTAD109,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-69839-002-0105,,ATOSU-58320-00017,44.37415,-124.95648,,194,580,
+RR1713,,,,CE04OSPS-PC01B-4B-PHSENA106,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-69839-002-0105,,ATAPL-58337-00003,44.37415,-124.95648,,194,580,
+RR1713,,,,CE04OSPS-PC01B-4D-PCO2WA105,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-69839-002-0105,,ATOSU-58336-00008,44.37415,-124.95648,,194,580,
+RR1713,,,,CE04OSPS-SF01B-2A-CTDPFA107,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-66662-00013,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-2A-DOFSTA107,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-58694-00007,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-2B-PHSENA108,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-58337-00011,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-3A-FLORTD104,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-58322-00013,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-3B-OPTAAD105,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-58332-00008,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-3C-PARADA102,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATAPL-66645-00004,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-3D-SPKIRA102,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-58341-00008,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-4A-NUTNRA102,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-68020-00008,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-4B-VELPTD106,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-70114-00008,44.37415,-124.95648,,,580,
+RR1713,,,,CE04OSPS-SF01B-4F-PCO2WA102,4,1,2017-07-29T05:00:00,2018-07-17T15:00:00,ATAPL-68870-002-0144,,ATOSU-58336-00007,44.37415,-124.95648,,,580,
+RR1811,,,,CE04OSPS-PC01B-05-ZPLSCB102,5,1,2018-07-17T15:00:00,,ATAPL-69839-002-0102,,ATOSU-63259-00001,44.37415,-124.95648,,194,580,
+RR1811,,,,CE04OSPS-PC01B-4A-CTDPFA109,5,1,2018-07-17T15:00:00,,ATAPL-69839-002-0102,,ATOSU-66662-00006,44.37415,-124.95648,,194,580,
+RR1811,,,,CE04OSPS-PC01B-4A-DOSTAD109,5,1,2018-07-17T15:00:00,,ATAPL-69839-002-0102,,ATOSU-58320-00007,44.37415,-124.95648,,194,580,
+RR1811,,,,CE04OSPS-PC01B-4B-PHSENA106,5,1,2018-07-17T15:00:00,,ATAPL-69839-002-0102,,ATAPL-58337-00008,44.37415,-124.95648,,194,580,
+RR1811,,,,CE04OSPS-PC01B-4D-PCO2WA105,5,1,2018-07-17T15:00:00,,ATAPL-69839-002-0102,,ATOSU-58336-00007,44.37415,-124.95648,,194,580,
+RR1811,,,,CE04OSPS-SF01B-2A-CTDPFA107,5,1,2018-07-17T15:00:00,,ATAPL-68870-002-0141,,ATOSU-66662-00007,44.37415,-124.95648,,,580,
+RR1811,,,,CE04OSPS-SF01B-2A-DOFSTA107,5,1,2018-07-17T15:00:00,,ATAPL-68870-002-0141,,ATOSU-58694-00004,44.37415,-124.95648,,,580,
```

Comment:
Here is the first error - is `ATOSU-58694-00004` mapping to `Sensor` in the bulk load-Sensors sheet?